specification.markdown

```diff
@@ -223,6 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |:---------------|:---------|:-----------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
+| parent         | Yes      | The path to the parent EBML Element. This path MUST be defined with the full hierarchy of EBML Elements separated with a `/`. The top EBML Element in the path hierarchy being the first in the value. Elements that can be found anywhere inside en EBML Element and its children MUST use the `*` (wildcard) value after the path of the first parent allowed. |
```

Comment:
Re:

> I'm not sure this is helpful. I don't oppose, but think that Parent Element alone is more flexible and less error-prone.

I'm not fond of having to specify the full path either, for the same reason. It also annoys the database designer in me as information isn't normalized and minimized.

Re:

> I would prefer to express multiple Parent Elements in this case. Such as, for `SimpleTag`, `parent="Tag|SimpleTag"`.

Me, too. Let's reserve `*` for "can occur anywhere". I have no strong feelings about keeping it an attribute with special syntax or using multiple child nodes for it.